lib/core/src/lib/viewer/components/viewer.component.html

```diff
@@ -120,6 +120,19 @@
                     </mat-menu>
                 </ng-container>
 
+                <ng-container *ngIf="allowGoBack && closeButtonPosition === 'right'">
+                    <adf-toolbar-divider></adf-toolbar-divider>
+
+                    <button class="adf-viewer-close-button adf-right-close-button"
```

Comment:
Same for the right class

Reply:
I used this class in ACA for style.

Reply:
Okay what about the other one?

Reply:
To make it in a sync I added for both. For now I am not using other one

Reply:
Okay so in this case please remove unused one

Reply:
I don't think that adding a class in ADF, and then defining its styles in ACA is the right approach.  It creates a tighter coupling, and would require us to raise PRs in ACA, everytime something over here changes. This would be even more difficult to maintain in cases like HX, where they are only using ADF, and not ACA

Reply:
Removed the classes and added the new check as **hideInfobutton**  which will hide and show the info button